scripts/werf-in-image/trdl.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine
+ARG groups
+ARG channels
+
+RUN apk add curl gnupg
+
+RUN curl -sSLO "https://tuf.trdl.dev/targets/releases/0.7.0/linux-amd64/bin/trdl" -O "https://tuf.trdl.dev/targets/signatures/0.7.0/linux-amd64/bin/trdl.sig" && \
+    curl -sSL https://trdl.dev/trdl.asc | gpg --import && \
+    gpg --verify trdl.sig trdl && \
+    rm trdl.sig && \
+    chmod +x ./trdl && \
+    mv trdl /usr/local/bin/trdl && \
+    trdl add werf https://tuf.werf.io 1 b7ff6bcbe598e072a86d595a3621924c8612c7e6dc6a82e919abe89707d7e3f468e616b5635630680dd1e98fc362ae5051728406700e6274c5ed1ad92bea52a2 && \
+    wget https://github.com/mikefarah/yq/releases/download/v4.24.2/yq_linux_amd64 -O /usr/local/bin/yq && \
+    chmod +x /usr/local/bin/yq
+
+COPY trdl_channels.yaml /trdl_channels.yaml
+
+RUN for group in ${groups}; do \
+        for channel in ${channels}; do \
+            echo "Perform trdl update for werf $group $channel ..." &&\
+            trdl update werf $group $channel && \
+            . $(trdl use werf $group $channel) && \
```

Comment:
The current implementation needs to check that we get the expected werf version for the channel (based on the `trdl_channels.yaml` file from repo) as before.